### PR TITLE
Release notes for 20.09

### PR DESCRIPTION
**Pull request by mvdbeek**

### CODE REVIEW

File: doc/source/releases/20.09.rst [hunk 1]
```diff
@@ -0,0 +1,687 @@
...
+.. bug_tag_tools
+
+* Allow submission of tools with unset hidden parameters
+  (thanks to `@bernt-matthias <https://github.com/bernt-matthias>`__).
+  `Pull Request 9656`_
+* Fix error in get_open_tempfile()
+  `Pull Request 10060`_
+* Backport bugfix in `#10060 <https://github.com/galaxyproject/galaxy/issues/10060>`__ to 20.05
+  `Pull Request 10138`_
+* Filtering.py failing all expressions
```
Comment (jdavcs) [inline doc/source/releases/20.09.rst]:
```suggestion
* Fix filtering.py failing all expressions
```
Why good commit messages matter.
(in release notes, this reads "Filtering.py failing all expressions (thanks to @...)"

File: doc/source/releases/20.09_announce_user.rst [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+.. figure:: images/20.09-upload-dropbox.png
+   :alt: Screenshot of Dropbox with a "bams" folder and several test files
+
+There will be a new button for "Choose Remote files", when sources are configured, and then you will be able to see and search across all of the remote data locations to which you might have access.
+
+.. figure:: images/20.09-upload-search.png
+   :alt: Search across sources. A modal dialog showing a new "Dropbox" folder amongst "FTP Directory" and library location.
+
+And then you will be able to see your load your files easily in Galaxy!
```
Comment (jdavcs) [inline doc/source/releases/20.09_announce_user.rst]:
```suggestion
You will be able to see and load your files easily in Galaxy!
```
(why -"And then": if you read this right after the previous paragraph, you get two consecutive "and then you will be able to see and...")

File: doc/source/releases/20.09_announce_user.rst [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+Improved Workflow Layout Algorithm
+----------------------------------
+
+The old automatic workflow layout algorithm liked to hide the flow of data within your workflow
+
+.. figure:: images/20.09-wf-before.png
+   :alt: Screenshot of a large workflow in Galaxy. Many connections go behind tools and are hard to follow.
+
+
+So we have replaced the layout algorithm with a newer one which should make the flow of data within your workflows easier to understand. You will still probably want to make manualy adjustments after the automatic layout, but hopefully this will save you some time.
```
Comment (jdavcs) [inline doc/source/releases/20.09_announce_user.rst]:
```suggestion
So we have replaced the layout algorithm with a newer one which should make the flow of data within your workflows easier to understand. You will still probably want to make manual adjustments after the automatic layout, but hopefully this will save you some time.
```

File: doc/source/releases/20.09_announce_user.rst [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+
+
+.. figure:: images/20.09-wf-after.png
+   :alt: Screenshot of a large workflow in Galaxy. Many connections go behind tools and are hard to follow.
+
+Workflow Reporting Improvements
+-------------------------------
+
+When one or more workflows is submitted, the status page reporting on those workflow(s) and their reports has been improved!
```
Comment (jdavcs) [inline doc/source/releases/20.09_announce_user.rst]: Maybe "The workflow status page and reports have been improved!"? Not sure if this preserves the meaning.
If not, how about "When one or more workflows **are** submitted..."?

File: doc/source/releases/20.09_announce_user.rst [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+  `Pull Request 10138`_
+* Fix and simplify built-in trim tool
+  (thanks to `@wm75 <https://github.com/wm75>`__).
+  `Pull Request 10145`_
+* Allow unary plus and minus in filtering
+  (thanks to `@wm75 <https://github.com/wm75>`__).
+  `Pull Request 10154`_
+* Toolbox refactoring, styling, and a couple bugfixes
+  `Pull Request 10170`_
+* Filtering.py failing all expressions
```
Comment (jdavcs) [inline doc/source/releases/20.09_announce_user.rst]:
```suggestion
* Fix filtering.py failing all expressions
```
Same as comment above.

File: doc/source/releases/20.09.rst [hunk 6]
```diff
@@ -0,0 +1,687 @@
...
+  `Pull Request 10189`_
+* Make cached container resolver pick newest build
+  (thanks to `@torfinnnome <https://github.com/torfinnnome>`__).
+  `Pull Request 10242`_
+* Don't run local runner job processes in an unnecessary shell
+  (thanks to `@nsoranzo <https://github.com/nsoranzo>`__).
+  `Pull Request 10257`_
+.. enhancement_tag_admin
+
+* It docs additional info
```
Comment (jdavcs) [inline doc/source/releases/20.09.rst:175]:
```suggestion
* IT docs additional info
```

**Comment by ic4f** [comment]
Considering all the "add/Added/adding.." in our release notes, maybe we should encourage commit message standards more proactively? (e.g., https://github.com/erlang/otp/wiki/writing-good-commit-messages). Maybe add some guidelines to `CONTRIBUTING.md`?

File: doc/source/releases/20.09_announce_user.rst (hunk 2 above)
Comment (hexylena) [reply doc/source/releases/20.09_announce_user.rst]: Just me completely forgetting what I wrote :sweat_smile: cheers

File: doc/source/releases/20.09_announce_user.rst (hunk 4 above)
Comment (hexylena) [reply doc/source/releases/20.09_announce_user.rst]: definitely should be are!
```suggestion
When one or more workflows are submitted, the status page reporting on those workflow(s) and their reports has been improved!
```

File: doc/source/releases/20.09.rst [hunk 7]
```diff
@@ -0,0 +1,687 @@
...
+
+* Use compression_utils to safely peek in compressed datasets
+  (thanks to `@pvanheus <https://github.com/pvanheus>`__).
+  `Pull Request 9990`_
+* fix OSError in fastq_to_fqtoc converter
+  (thanks to `@bgruening <https://github.com/bgruening>`__).
+  `Pull Request 10042`_
+* Don't display download link for optional, non-existant metadata files
+  `Pull Request 10068`_
+* Fix determining 'int', 'float' column type in Tabular
```
Comment (selten) [inline doc/source/releases/20.09.rst]:
```suggestion
* Fix determining 'int', 'float' column type in Tabular and BaseCSV
```

**Comment by mvdbeek** [comment]
> Maybe add some guidelines to `CONTRIBUTING.md`?

Can you draft something for this @ic4f ? It's certainly not a must but much nicer to read.

**Review by jdavcs — APPROVED** [review]
